### DISCUSSION TITLE
需要OpenWrt的mips_24kc版本

### DISCUSSION THREAD

**Discussion by kuangru52** [messense/aliyundrive-webdav#346] [Q&A] [answered]
请问能否提供OpenWrt 的mips_24kc版本，研究很长时间，始终不得要领啊

**Comment by messense** [comment, marked as answer]
一直都有：https://github.com/messense/aliyundrive-webdav/releases/download/v1.2.7/aliyundrive-webdav_1.2.7_mips_24kc.ipk

**Reply by kuangru52** [reply]
我发现我不仅是笨，还瞎
